src/test-c-gen/function_return_builder.cpp

```diff
@@ -0,0 +1,203 @@
+#include "function_return_builder.h"
+
+#include <test-c-gen/expr2cleanc.h>
+
+/*******************************************************************\
+Function: function_return_buildert::function_return_buildert
+Inputs:
+ all_inputs - All the inputs from the interpreter
```

Comment:
No, sorry, this comment seems misplaced. 
There was a more general question in some place how the interpreter knows what inputs and outputs are. Inputs and outputs of the function under test are marked in the goto trace.